package.json

```diff
@@ -76,11 +81,14 @@
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-react-hooks": "^4.5.0",
     "parcel": "^2.6.0",
+    "parcel-bundler": "^1.12.5",
```

Comment:
This dependency should be removed. You may need to merge or rebase the latest from master.